fe/fe-core/src/main/java/org/apache/doris/catalog/external/ExternalMetadataOperations.java

```diff
@@ -0,0 +1,40 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Move the this file and `[NamedExternalTable.java]` to package `datasource.operations`.
In future, all classes related to multi catalog are better in `datasource` package